### PR TITLE
Ov calculators compilation enabling

### PR DESCRIPTION
**Pull request by rasapala**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -165,6 +159,70 @@ http_archive(
...
+    srcs = glob(
+        [
+            paths.join(PREFIX, "lib/intel64/libopenvino_c.so"),
+            paths.join(PREFIX, "lib/intel64/libopenvino.so"),
+        ],
+    ),
+    hdrs = glob(
+        [
+            paths.join(PREFIX, "include/ie/**/*.h"),
+            paths.join(PREFIX, "include/**/*.hpp"),
```
Comment (atobiszei) [inline WORKSPACE:181]: I think just one header should suffice? Does any of the OV tensor converter calculators use different header than openvino.hpp?

File: src/BUILD [hunk 2]
```diff
@@ -355,6 +363,20 @@ cc_library(
     alwayslink = 1,
 )
 
+#OPENVINO CALCULATORS
+cc_binary(
```
Comment (atobiszei) [inline src/BUILD:367]: Maybe add "model" into target name to cleary distinguish that those are just model files

File: third_party/mediapipe_calculators/BUILD [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        "@google_mediapipe//mediapipe/calculators/core:matrix_to_vector_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:merge_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:stream_to_side_packet_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:constant_side_packet_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:graph_profile_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:get_vector_item_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:vector_size_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:packet_sequencer_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:merge_to_vector_calculator",
+        "@google_mediapipe//mediapipe/calculators/tflite:ssd_anchors_calculator",
```
Comment (atobiszei) [inline third_party/mediapipe_calculators/BUILD:82]: Why do we need tflite calculator? Is it used in POC?

File: third_party/mediapipe_calculators/BUILD [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        "@google_mediapipe//mediapipe/calculators/image:opencv_image_encoder_calculator",
+        "@google_mediapipe//mediapipe/calculators/image:opencv_put_text_calculator",
+        #GPU"@google_mediapipe//mediapipe/calculators/image:set_alpha_calculator",
+        #GPU"@google_mediapipe//mediapipe/calculators/image:bilateral_filter_calculator",
+        #GPU"@google_mediapipe//mediapipe/calculators/image:image_transformation_calculator",
+        "@google_mediapipe//mediapipe/calculators/image:image_cropping_calculator",
+        #GPU"@google_mediapipe//mediapipe/calculators/image:luminance_calculator",
+        #GPU"@google_mediapipe//mediapipe/calculators/image:sobel_edges_calculator",
+        #GPU"@google_mediapipe//mediapipe/calculators/image:recolor_calculator",
+        #ERROR"@google_mediapipe//mediapipe/calculators/image:scale_image_calculator",
```
Comment (atobiszei) [inline third_party/mediapipe_calculators/BUILD:100]: ?

File: third_party/opencv/install_opencv.sh [hunk 5]
```diff
@@ -54,7 +54,10 @@ fi
 current_working_dir=$(pwd)
 
 cd $work_dir
+rm -rf $opencv_branch $work_dir/opencv_repo
```
Comment (atobiszei) [inline third_party/opencv/install_opencv.sh:57]: Why do we need changes here?

**Review by atobiszei — APPROVED** [review]

File: third_party/mediapipe_calculators/BUILD (hunk 3 above)
Comment (rasapala) [reply third_party/mediapipe_calculators/BUILD:75]: Yes

File: third_party/mediapipe_calculators/BUILD (hunk 4 above)
Comment (rasapala) [reply third_party/mediapipe_calculators/BUILD]: Added comments.

**Pull request merged**